kuma/scrape/sources/base.py

```diff
@@ -144,9 +142,21 @@ def current_options(self):
 
     def decode_href(self, href):
         """Convert URL-escaped href attributes to unicode."""
-        decoded = unquote(binary_type(href))
-        assert isinstance(decoded, binary_type)
-        decoded = decoded.decode('utf8')
+        if PY3:  # pragma: no cover
+            # TODO: Remove the PY3 condition once we don't run Python 2 anymore
+            # In Python 3, unquote returns unicode
+            # decoded = unquote(href.decode('ascii'))
```

Comment:
It feels like this comment ``# decoded = unquote(href.decode('ascii'))`` is an earlier version of this code, and should be removed.